heypublisher-sub-mgr.php

```diff
@@ -271,151 +279,150 @@ function HeyPublisherAdminInit() {
 //   $hp_res->handler();
 // }
 
-/*
--------------------------------------------------------------------------------
-Initialize / Upgrade
--------------------------------------------------------------------------------
-*/
+// -----------------------------------------------------------------------------
+// Initialize / Upgrade
+// -----------------------------------------------------------------------------
 function heypub_init(){
-  global $hp_xml;
-
-  // we're referencing the old key names here directly - as we no longer need the defines in the system
-  // and this is simply for backwards upgrade compatibility
-  if (get_option('_heypub_opt_plugin_version_current') != false) {
-    // this user is upgrading from build version <= 28
-    $hp_xml->initialize_plugin();
-    // need to transfer over all of their install configs to the new hash and clean the db:
-    // plugin version and install date
-    $hp_xml->set_install_option('version_current',get_option('_heypub_opt_plugin_version_current'));
```

Comment:
removing all upgrade paths prior to 40.  We don't have any plugins still operating under that version.